lib/cmds/space_cmds/environment_cmds/create.js

```diff
@@ -0,0 +1,59 @@
+import { getContext } from '../../../context'
+import * as logging from '../../../utils/log'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+import { assertLoggedIn, assertSpaceIdProvided } from '../../../utils/assertions'
+
+export const command = 'create'
+
+export const desc = 'Create an environment'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful environment create --name \'Your Environment Name\'')
+    .option('environment-id', {
+      alias: 'e',
+      describe: 'Id of the environment to create',
+      demandOption: true
```

Comment:
Why is the id required? we can still create an environment with the base62 id, oder?